rust-connectors/sinks/sql/src/main.rs

```diff
@@ -40,7 +40,7 @@ async fn main() -> anyhow::Result<()> {
     let hub_url = raw_opts.hub_url;
     info!("hub_url={}", &hub_url);
     let mut transformations = Transformations::from_hub(hub_url, raw_opts.transform).await?;
-    info!("{} transformations loaded", transformations.size());
+    info!("{:?} transformations loaded", transformations);
```

Comment:
looks like we should keep size here or change to debug

Reply:
ah. Yeah. I'll change that to debug.